geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/NullRedisSortedSet.java

```diff
@@ -63,6 +63,11 @@ long zcount(SortedSetScoreRangeOptions rangeOptions) {
     return 0;
   }
 
+  @Override
```

Comment:
Again, alphabetizing is apparently what we're doing here.

Reply:
Done